README.md

```diff
@@ -52,65 +28,64 @@ tests, and also includes a number of useful features:
-serve the following purposes:
+As an example, we can look inside untangled-spec for the following files:
 
-- `all_tests.cljs` : An entry point for CI testing from the command line.
-- `suite.cljs` : The entry point for browser test rendering.
-- `tests_to_run.cljs` : A file that does nothing more than require all of the specs. The test runners search for
+- [all_tests.cljs](https://github.com/untangled-web/untangled-spec/blob/feature/documentation/test/untangled_spec/all_tests.cljs) : An entry point for CI testing from the command line.
+- [user.cljs](https://github.com/untangled-web/untangled-spec/blob/feature/documentation/dev/cljs/user.cljs) : The entry point for browser test rendering.
+- [tests_to_run.cljs](https://github.com/untangled-web/untangled-spec/blob/feature/documentation/test/untangled_spec/tests_to_run.cljs) : A file that does nothing more than require all of the specs. The test runners search for
 testing namespaces, so if you don't load them somewhere, they won't be found. Since there are two places tests
 run from (browser and CI) it makes sense to make this DRY.
 
-There is a `package.json` file for installing node packages to run CI tests.
-The `project.clj` includes various things to make all of this work:
+There is a [package.json](https://github.com/untangled-web/untangled-spec/blob/feature/documentation/package.json) file for installing node packages to run CI tests.
 
-- The lein doo plugin, for running tests through karma *via* node (in Chrome).
+The [project.clj](https://github.com/untangled-web/untangled-spec/blob/feature/documentation/project.clj) includes various things to make all of this work:
+- The [lein doo plugin](https://github.com/bensu/doo#usage), for running tests through karma *via* node (in Chrome).
 - A `:doo` section to configure the CI runner
-- A cljsbuild for test with figwheel true. This is the browser test build.
-- A cljsbuild for the CI tests output (automated-tests).
-- The lein `test-refresh` plugin, which will re-run server tests on save, and also can be configured with the
-spec renderer (see the `:test-refresh` section in the project file).
-
-## Running server tests
+- A cljsbuild for testing with figwheel true. The build with id "test" [here](https://github.com/untangled-web/untangled-spec/blob/feature/documentation/project.clj#L36), is the browser test build.
+- A cljsbuild for testing with lein doo. The build with id "automated-tests" [here](https://github.com/untangled-web/untangled-spec/blob/feature/documentation/project.clj#L47), is the CI tests output.
+- The [lein test-refresh plugin](https://github.com/jakemcc/lein-test-refresh), which will re-run server tests on save, and also can be configured with the spec renderer (see the `:test-refresh` section in the project file).
 
-See `test/server/app/server_spec.clj` for a sample specification (again, on the Tutorial project). To run all specs, just use:
+### Running server tests
```

Comment:
Not sure what I should do here then, do I talk about each necessary file? 
eg: `project.clj` `user.clj` `script/figwheel.clj` etc...